### DISCUSSION TITLE
Beginner's question: xrdp send images or drawing commands over network? compared to vnc, which use less traffic?

### DISCUSSION THREAD

**Discussion by myrfy001** [neutrinolabs/xrdp#2835] [General]
I'm asking this question because I want to save my traffic cost. Should I use VNC or xrdp? which use less traffic?

**Comment by matt335672** [comment]
The answer, as ever, is 'it depends'. Ultimately there's no replacement for measurement. Set up a local server on your LAN and try both.

Here are some thoughts for you:-
- Try to save on the amount of traffic being sent over the network. Avoid window effects such as fade-in and fade-out. Use a single colour background. Stick to a smaller display size for the remote session if you can.
- VNC uses the RFB protocol which is very simple - you won't be sending audio or complex clipboard data. Try tigervnc server and viewer as these give you a lot of knobs to play with to compress the data being sent. It's harder to tune xrdp in this way.
